examples/security-headers/public/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
should these files be in this PR?

Reply:
😬 Certainly not. Carried over from another branch. Will remove.